packages/moonstone/Scrollable/ScrollableNative.js

```diff
@@ -463,61 +463,44 @@ class ScrollableBaseNative extends Component {
 		}
 
 		const
-			{childRef, containerRef, scrollToAccumulatedTarget} = this.uiRef,
+			{childRef, containerRef} = this.uiRef,
 			bounds = this.uiRef.getScrollBounds(),
-			canScrollVertically = this.uiRef.canScrollVertically(bounds),
-			pageDistance = (isPageUp(keyCode) ? -1 : 1) * (canScrollVertically ? bounds.clientHeight : bounds.clientWidth) * paginationPageMultiplier,
 			spotItem = Spotlight.getCurrent();
 
-		if (spotItem) {
-			// Should skip scroll by page when spotItem is paging control button of Scrollbar
-			if (!childRef.containerRef.contains(spotItem)) {
-				return true;
-			}
-
+		// Should skip scroll by page when spotItem is paging control button of Scrollbar
+		if (spotItem && childRef.containerRef.contains(spotItem)) {
 			const
 				// VirtualList and Scroller have a spotlightId on containerRef
 				spotlightId = containerRef.dataset.spotlightId,
 				direction = this.getPageDirection(keyCode),
```

Comment:
Now, `keyCode` argument is only used to get `direction`, and `onKeyDown` gets a direction value just before calling `scrollByPage`. What about to change the argument to `direction'?